packages/cli/src/build/__tests__/build.ts

```diff
@@ -907,3 +907,227 @@ test("self import", async () => {
 
   `);
 });
+
+test("correct default export using mjs and dmts proxies", async () => {
+  let dir = await testdir({
+    "package.json": JSON.stringify({
+      name: "@mjs-proxy/repo",
+      preconstruct: {
+        packages: ["packages/pkg-a"],
+      },
+    }),
+    "packages/pkg-a/package.json": JSON.stringify({
+      name: "pkg-a",
+      main: "dist/pkg-a.cjs.js",
+      module: "dist/pkg-a.esm.js",
+      exports: {
+        ".": {
+          module: "./dist/pkg-a.esm.js",
+          import: "./dist/pkg-a.cjs.mjs",
```

Comment:
`.cjs.mjs` it's totally accurate but also 😂

Reply:
ye, i actually meant to mention this weirdo in a comment but forgot. I came to the same conclusion - it looks weird as hell but it's kinda OK. Of course, it also doesn't really matter but at the same time u can come up with a reasoning why it is like this :p

Reply:
We could also just completely avoid using `.mjs` and nest this file in a directory with `package.json: {type:'module'}` but I'm not sure if it's worth the added complexity 🤷

Reply:
Oh no, I think this is the right choice, it's just funny